MotorCommand command
float32 completion